src/main/java/org/opentripplanner/apis/gtfs/datafetchers/VehicleRentalStationImpl.java

```diff
@@ -96,6 +98,40 @@ public DataFetcher<Integer> vehiclesAvailable() {
     return environment -> getSource(environment).getVehiclesAvailable();
   }
 
+  @Override
+  public DataFetcher<Iterable<VehicleTypeCount>> vehicleTypesAvailable() {
+    return environment -> {
+      VehicleRentalPlace rentalPlace = getSource(environment);
+      if (rentalPlace instanceof VehicleRentalStation) {
+        VehicleRentalStation station = (VehicleRentalStation) rentalPlace;
+        return station.vehicleTypesAvailable
+          .entrySet()
+          .stream()
+          .map(vtc -> new VehicleTypeCount(vtc.getKey(), vtc.getValue()))
+          .toList();
+      } else {
+        return null;
+      }
+    };
+  }
+
+  @Override
+  public DataFetcher<Iterable<VehicleTypeCount>> vehicleSpacesAvailable() {
+    return environment -> {
+      VehicleRentalPlace rentalPlace = getSource(environment);
+      if (rentalPlace instanceof VehicleRentalStation) {
+        VehicleRentalStation station = (VehicleRentalStation) rentalPlace;
+        return station.vehicleSpacesAvailable
+          .entrySet()
+          .stream()
+          .map(vtc -> new VehicleTypeCount(vtc.getKey(), vtc.getValue()))
+          .toList();
+      } else {
+        return null;
+      }
+    };
+  }
+
   private VehicleRentalPlace getSource(DataFetchingEnvironment environment) {
```

Comment:
This type is incorrect. Since we explicit resolve a `VehicleRentalStation` (not a place) we can change it to that.

This means that further up, you don't need the `instanceOf`.